AI/Nullkiller/Engine/FuzzyEngines.cpp

```diff
@@ -65,7 +65,7 @@ armyStructure evaluateArmyStructure(const CArmedInstance * army)
 	for(auto s : army->Slots())
 	{
 		bool walker = true;
-		const CCreature * creature = s.second->type;
+		auto creature = s.second->getType()->accessBonuses();
```

Comment:
Why does `accessBonuses()` return a creature? This is confusing.

Reply:
renamed to bearer

Reply:
Will push after all other topics will decided)